data/scenarios/Challenges/flower-count.yaml

```diff
@@ -0,0 +1,177 @@
+version: 1
+name: Flower Count
+author: Brent Yorgey
+description: |
+  Count the flowers quickly... or else!
+creative: false
+objectives:
+  - id: count_flowers
+    teaser: Count the flowers!
+    goal:
+      - |
+        Your evil nemesis, the sadistic supervillain Dr. Nemesis, is
+        at it again!  They have wired up some explosives to your True
+        Love and lit the fuse, but promised to stop it... IF you can
+        count all the flowers in time!
+      - |
+        Specifically, Dr. Nemesis demands that you count the
+        total number of flowers in the 120x40 field bounded by walls,
```

Comment:
Let's motivate this objective with [arithmomania](https://en.wikipedia.org/wiki/Arithmomania#Folklore)?